bot/cogs/eval.py

```diff
@@ -124,7 +123,8 @@ def _format(self, inp, out):  # (str, Any) -> (str, discord.Embed)
 
         return res  # Return (text, embed)
 
-    async def _eval(self, ctx, code):  # (discord.Context, str) -> None
+    async def _eval(self, ctx: discord.Context, code: str) -> Optional[discord.Message]:
```

Comment:
This seems to be the last one, although this one doesn't prevent the bot from running (I think).

There's one other mention of `discord.Context` in the docstring of the `HelpSession` class, but that's not really within the scope of this PR.
```suggestion
    async def _eval(self, ctx: Context, code: str) -> Optional[discord.Message]:
```